### PR TITLE
Remove Advance throttling enable config

### PR DESCRIPTION
**Pull request by isharac**
Fix for https://github.com/wso2/product-apim/issues/8607

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/throttling/ThrottleDataHolder.java [hunk 1]
```diff
@@ -122,7 +122,7 @@ public boolean isAPIThrottled(String apiKey){
                 this.throttledAPIKeysMap.remove(apiKey);
                 this.conditionDtoMap.remove(apiKey);
                 return false;
-            }
+            }APIUtil
```
Comment (tharindu1st) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/throttling/ThrottleDataHolder.java:125]: remove APIUtil

**Comment by tharindu1st** [comment]
change the api-manager.xml.j2 as well
and verify profiles

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/throttling/ThrottleDataHolder.java (hunk 1 above)
Comment (isharac) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/throttling/ThrottleDataHolder.java]: fixed

**Comment by isharac** [comment]
> change the api-manager.xml.j2 as well
> and verify profiles

Tested and verified the profiles

**Pull request merged**